stats.go

```diff
@@ -118,8 +118,13 @@ func newStatsExporter(o Options, enforceProjectUniqueness bool) (*statsExporter,
 		vds := bundle.([]*view.Data)
 		e.handleUpload(vds...)
 	})
-	e.bundler.DelayThreshold = e.o.BundleDelayThreshold
-	e.bundler.BundleCountThreshold = e.o.BundleCountThreshold
+
+	if e.o.BundleDelayThreshold != 0 {
```

Comment:
I think https://github.com/census-instrumentation/opencensus-go/issues/177 mentions the default (zero value) is not doing a good job and we need to BundleDelayThreshold and BundleCountThreshold values to some hardcoded values if user set them to zero.

It might require some research to find the right values for the hardcoded values. Maybe this is not a good starter issue.

Reply:
Sounds good, you know best. 

I thought that with this change we *are* changing the default. I.e. when e.o.BundleDelayThreshold is 0, the value stays at bundler.DefaultDelayThreshold (as in the new test case I added).